FRW/Modules/Login/ViewModel/KeyStoreLoginViewModel.swift

```diff
@@ -207,7 +207,7 @@ final class KeyStoreLoginViewModel: ObservableObject {
     }
 }
 
-extension KeyStoreLoginViewModel {
+extension KeyStoreLoginViewModel_Old {
```

Comment:
no need keep `_Old` view model, we should keep one view model

Not need for keep one `old` viewmodel, one for `refactor` view model